### PR TITLE
Improvements to api-docs.yaml

### PR DESCRIPTION
**Pull request by curoli**
- Added additionalProperties where appropriate to signify a mapping property
- Added host, scheme
- Switched number to integer

### CODE REVIEW

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -10,6 +10,11 @@ info:
     url: http://opensource.org/licenses/BSD-3-Clause
   termsOfService: https://github.com/broadinstitute/firecloud-orchestration
 
+host: api.firecloud.org
```
Comment (MatthewBemis) [inline src/main/resources/swagger/api-docs.yaml]: this change causes some issues when using this on non-production environments as all calls from the swagger on non-prod environments will be sent to production orchestration.

we would likely need to template this in somehow in order to get the desired behavior. i.e., we have a mapping of environments to hosts:

local -> local.broadinstitute.org:10443
dev -> firecloud-orchestration.dsde-dev.broadinstitute.org
alpha -> firecloud-orchestration.dsde-alpha.broadinstitute.org
firecloud in a box (dev) -> firecloud-fiab.dsde-dev.broadinstitute.org:23443
firecloud in a box (qa) -> firecloud-fiab.dsde-qa.broadinstitute.org:23443
...and so on...
prod -> api.firecloud.org

**Review by MatthewBemis — CHANGES_REQUESTED** [review]

File: src/main/resources/swagger/api-docs.yaml (hunk 1 above)
Comment (curoli) [reply src/main/resources/swagger/api-docs.yaml]: OK, removed host and schemes again.

**Review by MatthewBemis — APPROVED** [review]

**Comment by curoli** [comment]
Thank you for approving! Could some one merge it? I don't have the necessary permissions. Thanks!

**Comment by MatthewBemis** [comment]
@curoli sure, I can merge it. thanks for contributing improvements!

**Pull request merged**